oak-api/src/main/java/org/apache/jackrabbit/oak/api/jmx/IndexStatsMBean.java

```diff
@@ -127,6 +128,15 @@ public interface IndexStatsMBean {
      */
     String getReferenceCheckpoint();
 
+    @Description("Force update the indexing lane to a latest checkpoint. This will abort and pause the running lane, release its lease and set the reference checkpoint to a latest one." +
```

Comment:
Can you be more specific of what latest mean? Exact values if possible.

Reply:
it would be to a checkpoint created during this method execution - I will update that.